pkg/components/ytsaurus_client.go

```diff
@@ -317,45 +311,12 @@ func (yc *YtsaurusClient) handleUpdatingState(ctx context.Context) (ComponentSta
 
 	case ytv1.UpdateStateWaitingForSnapshots:
 		if !yc.ytsaurus.IsUpdateStatusConditionTrue(consts.ConditionSnaphotsSaved) {
-			if !yc.ytsaurus.IsUpdateStatusConditionTrue(consts.ConditionSnapshotsMonitoringInfoSaved) {
-				monitoringPaths, err := yc.GetMasterMonitoringPaths(ctx)
-				if err != nil {
-					return SimpleStatus(SyncStatusUpdating), err
-				}
-				yc.ytsaurus.GetResource().Status.UpdateStatus.MasterMonitoringPaths = monitoringPaths
-
-				yc.ytsaurus.SetUpdateStatusCondition(ctx, metav1.Condition{
-					Type:    consts.ConditionSnapshotsMonitoringInfoSaved,
-					Status:  metav1.ConditionTrue,
-					Reason:  "Update",
-					Message: "Snapshots monitoring info saved",
-				})
-			}
-
-			if !yc.ytsaurus.IsUpdateStatusConditionTrue(consts.ConditionSnapshotsBuildingStarted) {
-				dnds, err := yc.getDataNodesInfo(ctx)
-				if err != nil {
-					return SimpleStatus(SyncStatusUpdating), err
-				}
-				log.FromContext(ctx).Info("data nodes before snapshots building", "dataNodes", dnds)
-				if err := yc.StartBuildMasterSnapshots(ctx, yc.ytsaurus.GetResource().Status.UpdateStatus.MasterMonitoringPaths); err != nil {
-					return SimpleStatus(SyncStatusUpdating), err
-				}
-
-				yc.ytsaurus.SetUpdateStatusCondition(ctx, metav1.Condition{
-					Type:    consts.ConditionSnapshotsBuildingStarted,
-					Status:  metav1.ConditionTrue,
-					Reason:  "Update",
-					Message: "Snapshots building started",
-				})
-			}
-
-			built, err := yc.AreMasterSnapshotsBuilt(ctx, yc.ytsaurus.GetResource().Status.UpdateStatus.MasterMonitoringPaths)
+			dnds, err := yc.getDataNodesInfo(ctx)
 			if err != nil {
 				return SimpleStatus(SyncStatusUpdating), err
 			}
-
-			if !built {
+			log.FromContext(ctx).Info("data nodes before snapshots building", "dataNodes", dnds)
```

Comment:
"what data nodes"?

Let's issue k8s events when it starts and finishes.
This is pretty much important thing and it should be visible on top level.